src/modules/tests/test-data/cat-d/safety-questions/__tests__/safety-questions.cat-d.selector.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { CatDUniqueTypes } from '@dvsa/mes-test-schema/categories/D';
+import { safetyQuestionsExist } from '../safety-questions.cat-d.selector';
+
+describe('Safety Questions Selector Cat D', () => {
+
+  describe('safetyQuestionsExist', () => {
+    it('should return false if there are no safety questions entered', ()  => {
+      const emptySafetyQuestions: CatDUniqueTypes.SafetyQuestions = { questions: [{}], faultComments: '' };
+      const result = safetyQuestionsExist(emptySafetyQuestions);
+      expect(result).toBeFalsy();
+    });
+    it('should return true if there are safety checks entered', ()  => {
+      const populatedSafetyQuestions: CatDUniqueTypes.SafetyQuestions =
+        {
+          questions: [{ description: 'test s01', outcome: 'P' }],
+          faultComments: '',
+        };
+      const result = safetyQuestionsExist(populatedSafetyQuestions);
+      expect(result).toBeTruthy();
+    });
+    it('should return false if there are vehicle checks but no outcome selected', ()  => {
```

Comment:
should be safety checks not vehicle checks.

Reply:
Good spot - thanks!